src/Entity/Article.php

```diff
@@ -0,0 +1,154 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\ArticleRepository;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+use Gedmo\Mapping\Annotation as Gedmo;
+
+/**
+ * @ORM\Entity(repositoryClass=ArticleRepository::class)
+ */
+class Article
+{
+    /**
+     * @ORM\Id
+     * @ORM\GeneratedValue
+     * @ORM\Column(type="integer")
+     */
+    private $id;
+
+    /**
+     * @ORM\Column(type="string", length=255, nullable=true)
+     */
+    private $title;
+
+    /**
+     * @ORM\Column(type="string", length=255, nullable=true)
+     */
+    private $body;
+
+    /**
+     * @ORM\Column(type="string", length=255, nullable=true)
+     */
+    private $inserted_by;
```

Comment:
The coding standard is to use camelCase for function and variable names. https://github.com/FullSpeedInc/admatrix_admin_system_docs/blob/master/documents_lts/team/coding_guidelines/php_coding_conventions.md